pulp_rpm/app/models.py

```diff
@@ -200,6 +203,184 @@ class SrpmContent(RpmContent):
     TYPE = 'srpm'
 
 
+class UpdateRecord(Content):
+    """
+    The "UpdateRecord" content type. Formerly "Errata".
```

Comment:
s/Errata/Erratum/ ?

Reply:
Nope, referring to the actual model name

 https://github.com/pulp/pulp_rpm/blob/2-master/plugins/pulp_rpm/plugins/db/models.py#L1092

Reply:
Ah, can you maybe clarify this comment then by saying something like 'Formerly "Errata" in Pulp 2'?